developer-docs-site/docs/nodes/validator-node/operator/update-validator-node.md

```diff
@@ -11,6 +11,16 @@ Since you are already running [a validator node and a validator fullnode](node-r
 
 This page explains how to make this swap, which largely amounts to switching out files and configuration settings between the two nodes. For a community-provided version of this document for Docker setup, see [Failover and migrate Validator Nodes for less downtime](https://forum.aptoslabs.com/t/failover-and-migrate-validator-nodes-for-less-downtime/144846).
 
+## Secure
```

Comment:
I'm not sure if "Secure" is the right title 🤔 I'd probably have a new section at the bottom of this document, where the title is something like "Running multiple Validator Fullnodes".

Reply:
Thanks! I did this and also rewrote the intro a bit to match the flow.